v6/technical/filtering-generic.md

```diff
@@ -0,0 +1,203 @@
+# Generic filtering
+
+## Implementation notes
+
+- Filtering support is **optional**. Implementing organisations **MAY** support it fully,
+  partially, or not at all. Support for filtering **SHOULD** be documented in the service
+  endpoint description. Support for this functionality **cannot be enforced** by any
```

Comment:
This is not yet part of the service endpoint. Shouldn't we have this be part of the specification?